src/algorithm/HISTOGRAM.hpp

```diff
@@ -91,14 +91,15 @@ class HISTOGRAM : public KernelBase
   void updateChecksum(VariantID vid, size_t tune_idx);
   void tearDown(VariantID vid, size_t tune_idx);
 
-  void runSeqVariant(VariantID vid, size_t tune_idx);
-  void runOpenMPVariant(VariantID vid, size_t tune_idx);
-  void runCudaVariant(VariantID vid, size_t tune_idx);
-  void runHipVariant(VariantID vid, size_t tune_idx);
-  void runOpenMPTargetVariant(VariantID vid, size_t tune_idx);
-
-  void setCudaTuningDefinitions(VariantID vid);
-  void setHipTuningDefinitions(VariantID vid);
+  void defineSeqVariantTunings();
```

Comment:
In some kernel headers, like this one, we have define*Tunings() methods grouped together and the run*Variant() methods grouped together below.  In others, the define and run methods are interleaved. I think we should pick one organization (I prefer the former) and apply it uniformly across all header files.

This would make the overall code easier to read and to see what's implemented for each kernel.